call-for-adoption-process.md

```diff
@@ -0,0 +1,11 @@
+* this process is meant for proposals that are candidate for adoption by the WebRTC Working Group, developed in their own repo (which typically should NOT be a @w3c repo); the said proposals are expected to be marked as CG report or Unofficial Drafts until their formal WG adoption
+
+* when the Call for Adoption (CfA) starts, and until its review period ends (typically 7 days) , the repo main branch is frozen
+
+* changes that would emerge from the CfA discussion would be filed as pull requests; these pull requests can be submitted by the reviewer asking for change with prompt reviews by the proposer, or vice versa
```

Comment:
voice: leaving oult "would" makes the text easier to read: "Changes that emerge from the CfA discussion, if any, must be filed as pull requests".